plugins/base/src/test/kotlin/resourceLinks/ResourceLinksTest.kt

```diff
@@ -205,4 +207,93 @@ class ResourceLinksTest : BaseAbstractTest() {
             }
         }
     }
+
+    @Test
+    fun `should not add unknown resources to the end of the head section`() {
```

Comment:
Additionally, you can check `head().children` are only tags, no text block.

Reply:
Nice idea! I definitely like it more than my current test.

Re-wrote it to check that there are no text nodes